Baby Monitor/Source Files/Custom Modules/View Controllers/TypedPageViewController.swift

```diff
@@ -19,7 +19,7 @@ internal class TypedPageViewController<BackgroundView: UIView>: BasePageViewCont
     ///     - analyticsScreenType: The type of the screen in terms of analytics tracking.
     init(viewMaker: @escaping @autoclosure () -> BackgroundView,
          analytics: AnalyticsManager,
-         analyticsScreenType: AnalyticsScreenType){
+         analyticsScreenType: AnalyticsScreenType) {
```

Comment:
Are you sure that your version of formatting doesn't trigger warning? From what I remember it should.
```suggestion
         analyticsScreenType: AnalyticsScreenType
    ) {
```

Reply:
It doesn't :) Do you remember which rule triggers it?

Reply:
`multiline_parameters_brackets `

Reply:
This rule is in the list of disabled :)